### PR TITLE
docs: add doc(en) for express adapter package

### PR DESCRIPTION
**Pull request by ShenChen93**
add doc(en) for express adapter package
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Docs(Add new document content)
- [ ] Translate Docs(Translate document content)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [x] I have read the **README** document
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

[issues]: https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/issues
[license]: http://aws.amazon.com/apache2.0/
[cla]: http://en.wikipedia.org/wiki/Contributor_License_Agreement

### CODE REVIEW

File: docs/en/Host-Skill-As-Webservice.rst [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+The web service must meet certain requirements to handle requests sent by Alexa
+and adhere to the Alexa Skills Kit interface standards. For more information,
+see
+`Host a Custom Skill as a Web Service <https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-a-web-service.html>`__
+in the Alexa Skills Kit technical documentation.
+
+.. warning::
+
+    These features are currently in beta. You can view the source
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: Why is this marked as beta?

File: docs/en/Host-Skill-As-Webservice.rst [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    const app = express();
+    const skillBuilder = Alexa.SkillBuilders.custom();
+    const skill = skillBuilder.create();
+    const adapter = new ExpressAdapter(skill, true, true);
+
+    app.post('/', adapter.getRequestHandlers());
+    app.listen(3000);
+
+
+For web application without express framework
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: Nit: inconsistent caps throughout. Should be Express

File: docs/en/Host-Skill-As-Webservice.rst [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+.. tabs::
+
+  .. code-tab:: javascript
+    const Alexa = require('ask-sdk-core');
+    const { SkillRequestSignatureVerifier, TimestampVerifier } = require('ask-sdk-express-adapter');
+
+    const skillBuilder = Alexa.SkillBuilders.custom();
+    const skill = skillBuilder.create();
+
+    // In your http server, suppose you already get request body in Json format
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: We may want to reword this.

File: docs/en/Host-Skill-As-Webservice.rst [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    // and htpp request headers
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: HTTP

File: docs/en/Host-Skill-As-Webservice.rst [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+---------------------------
+
+The Alexa Skills Kit SDK (ASK SDK) for Node provides boilerplate code for 
+request and timestamp verification through the
+`ask-sdk-express-adapter <https://www.npmjs.com/package/ask-sdk-express-adapter/>`__
+package. This package provides the verification components by exporting
+the `SkillRequestSignatureVerifier class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/verifier/index.ts#L56/>`__
+and `TimestampVerifier class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/verifier/index.ts#L292/>`__.
+Also this package provides `ExpressAdapter class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/adapter/ExpressAdapter.ts#L24/>`__ 
+which assembles the verification ability and skill invokation ability,
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: *invocation

File: docs/en/Host-Skill-As-Webservice.rst (hunk 1 above)
Comment (ShenChen93) [reply docs/en/Host-Skill-As-Webservice.rst]: Removed

File: docs/en/Host-Skill-As-Webservice.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+Host a Custom Skill as a Web Service
+====================================
+
+You can build a custom skill for Alexa by implementing a web service that
+accepts requests from and sends responses to the Alexa service in the cloud.
+
+The web service must meet certain requirements to handle requests sent by Alexa
+and adhere to the Alexa Skills Kit interface standards. For more information,
+see
+`Host a Custom Skill as a Web Service <https://developer.amazon.com/docs/custom-skills/host-a-custom-skill-as-a-web-service.html>`__
```
Comment (RonWang) [inline docs/en/Host-Skill-As-Webservice.rst]: This name is same as the tilte, which can be confusing when reading the doc, quoting itself in the doc. I know this doc is pointing to developer doc, but maybe better to rename either this or title

File: docs/en/Host-Skill-As-Webservice.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+The Alexa Skills Kit SDK (ASK SDK) for Node provides boilerplate code for 
+request and timestamp verification through the
+`ask-sdk-express-adapter <https://www.npmjs.com/package/ask-sdk-express-adapter/>`__
+package. This package provides the verification components by exporting
+the `SkillRequestSignatureVerifier class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/verifier/index.ts#L56/>`__
+and `TimestampVerifier class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/verifier/index.ts#L292/>`__.
+Also this package provides `ExpressAdapter class <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/adapter/ExpressAdapter.ts#L24/>`__ 
+which assembles the verification ability and skill invocation ability,
+make it easier to register requestHandlers on your `express application <http://expressjs.com/en/5x/api.html#app/>`__.
```
Comment (RonWang) [inline docs/en/Host-Skill-As-Webservice.rst]: syntax, either `makes` or `making`

File: docs/en/Host-Skill-As-Webservice.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+Installation
+~~~~~~~~~~~~
+
+You can install the ``ask-sdk-express-adapter`` package through NPM:
```
Comment (RonWang) [inline docs/en/Host-Skill-As-Webservice.rst]: Maybe better to use `You can install the package to your skill project`

File: docs/en/Host-Skill-As-Webservice.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+
+For web application with express framework
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `ExpressAdapter <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/adapter/ExpressAdapter.ts#L24/>`__
+class registers the skill instance from
+the ``SkillBuilder`` object, and provides a ``getRequestHandlers``
+method that return arrys of request handlers which can be registed
+to your express application
```
Comment (RonWang) [inline docs/en/Host-Skill-As-Webservice.rst]: miss a period

File: docs/en/Host-Skill-As-Webservice.rst [hunk 10]
```diff
@@ -0,0 +1,125 @@
...
+
+
+For web application with express framework
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The `ExpressAdapter <https://github.com/alexa/alexa-skills-kit-sdk-for-nodejs/blob/2.0.x/ask-sdk-express-adapter/lib/adapter/ExpressAdapter.ts#L24/>`__
+class registers the skill instance from
+the ``SkillBuilder`` object, and provides a ``getRequestHandlers``
+method that return arrys of request handlers which can be registed
+to your express application.
```
Comment (breedloj) [inline docs/en/Host-Skill-As-Webservice.rst]: *Express

**Review by breedloj — DISMISSED** [review]

**Review by RonWang — APPROVED** [review]
Shippppppp

**Review by breedloj — APPROVED** [review]

**Pull request merged**